roles/identity-management/manage-rh-sso-identities/tasks/delete_users.yml

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
Doesn't look like we're actually doing anything with `delete_users.yml` yet, so do we just want to remove this for now? If we want to keep it in this first iteration, how are we noting between users that we want created and users we want deleted? Right now it looks like we're just commenting things out which isn't the direction that I think we'd want to go.

Reply:
Makes sense. Originally, I wanted to make use of tags to differentiate one operation from another, but we didn't have the chance to discuss this particular piece in detail. We can either pursue this or remove it. What would be your preference?